### PR TITLE
benchmark npm packages directly

### PR DESCRIPTION
**Pull request by garyjN7**
benchmark.js has been changed to allow the installation of node packages and run benchmark tests against files in the package. This will allow the benchmark tests to be easily ran against a wider range of projects.

### CODE REVIEW

**Comment by hzoo** [comment]
Nice!

File: scripts/benchmark.js [hunk 1]
```diff
@@ -12,14 +12,28 @@ const babel  = require("babel-core");
 const zlib   = require("zlib");
 const fs     = require("fs");
 const path   = require("path");
+const Command = require("commander").Command;
```
Comment (hzoo) [inline scripts/benchmark.js]: I don't think we are using commander?

Comment (garyjN7) [inline scripts/benchmark.js]: Sorry, I didn't realize commander was pulled in by another package.  Have added commander to devDependencies in package.json

**Comment by kangax** [comment]
@garyjN7 could you please rebase

**Comment by hzoo** [comment]
we can merge it manually

**Comment by hzoo** [comment]
Merged via https://github.com/babel/babili/issues/271 - thanks a lot @garyjN7!

**Pull request closed without merging**